src/App/integrationTests/collectRecords/observersInput.test.jsx

```
@@ -25,25 +25,26 @@ test('Observers input shows users that have been removed from the project and al
 
   await waitFor(() =>
-    expect(observersRow).toHaveTextContent(
-      'Betsy Craig is an observer on this sample unit but is no longer a part of this project.',
-    ),
+    expect(within(observersRow).getByTestId('removed-observer-warning')).toBeInTheDocument(),
   )
-  fireEvent.click(within(observersRow).getByRole('button', { name: 'Remove as observer' }))
+  const removeObserverButton = within(observersRow).getByTestId('remove-observer-button')
 
-  const modal = screen.getByLabelText('Remove observer from record')
+  await user.click(removeObserverButton)
```

ESLint

[error] Avoid direct Node access. Prefer using the methods from Testing Library.

(testing-library/no-node-access)